package.json

```diff
@@ -41,6 +42,8 @@
     "typescript": "^3.6.2"
   },
   "dependencies": {
+    "@azure/arm-apimanagement": "^5.1.1",
+    "@azure/ms-rest-nodeauth": "^2.0.5",
```

Comment:
The latest version of `@azure/ms-rest-nodeauth` is `3.0.3`, but it issues a type mismatch when used with ` @azure/arm-apimanagement`. This bug has already been reported in [this issue](https://github.com/Azure/azure-sdk-for-js/issues/7258#issuecomment-582759072).